python/GafferUI/PathListingWidget.py

```diff
@@ -587,7 +584,6 @@ def __keyPress( self, widget, event ) :
 	# for compatibility with `GafferUI.Widget.dragBeginSignal()`.
 	def __buttonPress( self, widget, event ) :
 
-		self.__updateSelectionInButtonRelease = False
```

Comment:
I haven't reviewed this commit thoroughly, as I think it is problematic. I think it is the reason that it's no longer possible to drag a multiple selection from the hierarchy view. The click to start the drag is immediately deselecting all the other paths...